src/Row.ts

```diff
@@ -0,0 +1,47 @@
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import { RegistryMixin, RegistryMixinProperties } from '@dojo/widget-core/mixins/Registry';
+import { v, w } from '@dojo/widget-core/d';
+import { HasColumns } from './interfaces';
+import { CellProperties } from './Cell';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+
+import * as rowClasses from './styles/row.css';
+
+export interface RowProperties extends WidgetProperties, HasColumns, RegistryMixinProperties {
+	item: any;
```

Comment:
Could there be any benefit from making this generic, or at least providing a base type? It looks like the code is assuming that `item.data` will exist for example. 

The `field` properties in the column definitions could even be required to be a member of `keyof ItemDataType` or something.